posthog/temporal/common/posthog_client.py

```diff
@@ -12,6 +12,18 @@
     WorkflowInterceptorClassInput,
 )
 
+SAFE_INPUTS = (
+    "team_id",
+    "batch_export_id",
+    "interval",
+    "data_interval_start",
+    "data_interval_end",
+    "exclude_events",
+    "include_events",
+    "backfill_details",
+    "batch_export_model",
+)
```

Comment:
Can we add all the values from `ExternalDataWorkflowInputs`:

```
team_id: int
external_data_source_id: uuid.UUID
external_data_schema_id: uuid.UUID | None
billable: bool
reset_pipeline: Optional[bool]
```

Reply:
yes, certainly! I'm actually going to refactor the way this works after discussing with Tomas on Slack. I'm going to add a way to annotate the inputs dataclass so you can specify which fields are safe to log.